src/components/DataTable/index.js

```diff
@@ -1,9 +1,4 @@
-import Cell from './Cell';
-import Row from './Row';
-import Table from './Table';
-
-export {
-  Cell,
-  Row,
-  Table
-};
+export { default as DataTableColumn } from './DataTableColumn';
```

Comment:
Please use `import` and `export` here. This syntax leads to the PropTypes and PropDescriptions not being picked up in Storybook.

Reply:
👍